src/style.css

```
@@ -1 +1,81 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+
+  width: 40px;
+  height: 40px;
+}
+
+.menu-list {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+
+  list-style: none;
+
+  margin: 0;
+  padding: 0;
+}
+
+.menu-item {
+  position: relative;
+}
+
+.menu-item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.menu_links {
+  display: flex;
+  align-items: center;
+
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+
+  height: 60px;
+}
+
+.menu_links:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+a.is-active::after {
```

Don't style using tags. You have class, use it.
```suggestion
.is-active::after {
```